### ISSUE TITLE
[Question] 请问7B没有用上FlashAttention吗？

### ISSUE THREAD

**Issue by nezhazheng** [baichuan-inc/baichuan-7B#130]
### Required prerequisites

- [X] I have read the documentation <https://github.com/baichuan-inc/baichuan-7B/blob/HEAD/README.md>.
- [X] I have searched the [Issue Tracker](https://github.com/baichuan-inc/baichuan-7B/issues) and [Discussions](https://github.com/baichuan-inc/baichuan-7B/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [X] Consider asking first in a [Discussion](https://github.com/baichuan-inc/baichuan-7B/discussions/new).

### Questions

请问7B没有用上FlashAttention吗？看了下7B代码，没发现这块的逻辑。

### Checklist

- [X] I have provided all relevant and necessary information above.
- [X] I have chosen a suitable title for this issue.

**Comment by mmmans** [comment]
No.  We user xformers for training, and naive impl for inference.
